### DISCUSSION TITLE
2024/11/26/%E9%83%BD2024%E5%B9%B4%E6%9C%AB%EF%BC%8C%E9%A9%AC%E4%B8%8A2025%E5%B9%B4%E4%BA%86%EF%BC%8C%E8%BF%98%E6%9C%89%E4%BA%BA%E7%8E%A9%E7%88%B94%E5%90%97%EF%BC%9F/

### DISCUSSION THREAD

**Discussion by giscus** [Torae233/Torae233.github.io#8] [Announcements]
# 2024/11/26/%E9%83%BD2024%E5%B9%B4%E6%9C%AB%EF%BC%8C%E9%A9%AC%E4%B8%8A2025%E5%B9%B4%E4%BA%86%EF%BC%8C%E8%BF%98%E6%9C%89%E4%BA%BA%E7%8E%A9%E7%88%B94%E5%90%97%EF%BC%9F/

一个热爱音乐，爱玩FPS和Galgame的无产阶级革命者

https://torae233.github.io/2024/11/26/%E9%83%BD2024%E5%B9%B4%E6%9C%AB%EF%BC%8C%E9%A9%AC%E4%B8%8A2025%E5%B9%B4%E4%BA%86%EF%BC%8C%E8%BF%98%E6%9C%89%E4%BA%BA%E7%8E%A9%E7%88%B94%E5%90%97%EF%BC%9F/

**Comment by Torae233** [comment]
这是一条测试giscus评论系统的评论

**Comment by HU9050** [comment]
这是一条测试giscus评论系统的评论
